script.js

```diff
@@ -0,0 +1,155 @@
+const mobileMenu = document.getElementById('mobile-menu');
+const menu = document.getElementById('menu');
+const close = document.getElementById('close-btn');
+const mobileLinks = document.querySelectorAll('.nav-list');
+
+function openMenu() {
+  mobileMenu.style.display = 'flex';
+}
+
+function closeMenu() {
+  mobileMenu.style.display = 'none';
+}
+
+menu.addEventListener('click', openMenu);
+close.addEventListener('click', closeMenu);
+
+mobileLinks.forEach((link) => {
+  link.addEventListener('click', closeMenu);
+});
+
+const speakerData = [
+  {
+    img: './images/speaker_01.png',
```

Comment:
- [ ] wonderful job. Please, like mentioned in your review guidelines, You should personalize the content of your page. The design should be followed, but content changed to suit your topic. I can see the speaker section is the exact same thing with the original design. Please kindly change the pictures, and contents of the speakers. Consider this a means to learn to be more creative.
- [ ] kindly reduce the middle gap between the speakers so as to follow the design.
- [ ] please kindly add the image size as well
yours
![Screenshot (1095)_LI](https://github.com/erickma1/CapstoneProject/assets/65298048/d277db3a-70c5-4db8-8043-12c3b506de4a)
original
![Screenshot (1095)](https://github.com/erickma1/CapstoneProject/assets/65298048/09b898cb-d4dd-46a6-ab0e-1f38b2e58c69)